assets/agenda/components/AgendaList.tsx

```diff
@@ -428,7 +428,7 @@ class AgendaList extends React.Component<IProps, IState> {
                 }}
                 onScroll={this.props.onScroll}
             >
-                {this.props.groupedItems.map((group) => (
+                {this.props.groupedItems.filter((group) => group.items.length > 0).map((group) => (
```

Comment:
I think this might cause an issue to not show the `hiddenItems` for dates where no other `items` are shown.

Example:
```
Date 1:
items: 1
hiddenItems: 1

Date 2:
items: 0
hiddenItems: 1

Date 3:
items: 1
hiddenItems: 1

Date 4:
items: 0
hiddenItems: 1
```

With this change `Date 2` would not be shown, which I believe may cause confusion.

What we could do instead is use the date of the last group that has items to be shown (disregarding hidden items) to determine which is the last date in the list that should be shown. As the list of items received from the server are paginated and sorted by date, this might suffice. What do you think?

In the above example, the last shown date would be for `Date 3` (as that is the last group with an item count), which would show `Date 2` but not `Date 4`.